server/svix-server/src/core/security.rs

```diff
@@ -248,6 +248,14 @@ pub fn generate_org_token(keys: &Keys, org_id: OrganizationId) -> Result<String>
     Ok(keys.key.authenticate(claims).unwrap())
 }
 
+pub fn generate_management_token(keys: &Keys) -> Result<String> {
+    let claims =
+        Claims::with_custom_claims(CustomClaim { organization: None }, Duration::from_mins(10))
+            .with_issuer(JWT_ISSUER)
+            .with_subject("org_0000SvixManagement0000");
```

Comment:
Make the subject a constant. Btw, is it a valid ksuid? Does `0000SvixManagement0000` actually works as a ksuid and you can get the timestamp, etc from it?